Lets Do This/Views/Campaign/LDTCampaignDetailCampaignCell.m

```diff
@@ -95,11 +107,35 @@ - (void)setCoverImageURL:(NSURL *)coverImageURL {
     }];
 }
 
-- (void)setDisplaySubmitReportbackButton:(BOOL)displaySubmitReportbackButton {
-    _displaySubmitReportbackButton = displaySubmitReportbackButton;
-    if (displaySubmitReportbackButton) {
-        // @todo: Create public SubmitReportbackButtonTitle property.
-        [self.submitReportbackButton setTitle:@"Prove it".uppercaseString forState:UIControlStateNormal];
+- (void)setDisplayCampaignDetailsView:(BOOL)displayCampaignDetailsView {
+    _displayCampaignDetailsView = displayCampaignDetailsView;
+    if (!_displayCampaignDetailsView) {
+        self.diagonalShapeLayer.hidden = YES;
+        // Hack to avoid UILabels preventing  campaignDetailsView from fully collapsing.
```

Comment:
References comment in http://stackoverflow.com/a/18066138/1470725: "Good answer. Although not what I wanted to hear. I simply set the label to an empty string"

Not sure if this could possibly be causing the breaking constraints though when you start from the Presignup view (and submit a reportback right after, in order to break constraints)